bulkExport/extractPatientCompartmentSearchParams.js

```diff
@@ -0,0 +1,49 @@
+const fs = require('fs');
+const compartmentPatientV3 = require('./schema/compartmentdefinition-patient.3.0.2.json');
+const compartmentPatientV4 = require('./schema/compartmentdefinition-patient.4.0.1.json');
+const baseSearchParamsV3 = require('../../fhir-works-on-aws-search-es/src/schema/compiledSearchParameters.3.0.1.json');
+const baseSearchParamsV4 = require('../../fhir-works-on-aws-search-es/src/schema/compiledSearchParameters.4.0.1.json');
+
+// Create a dictionary of search params
+function extractPatientCompartmentSearchParams(baseSearchParams, compartmentPatient) {
```

Comment:
Add comments describing how to use this script. Similar to https://github.com/awslabs/fhir-works-on-aws-search-es/blob/mainline/scripts/searchParamsCompiler/run.ts#L7-L21

Make sure to note on the comments where does the `bulkExport/schema/compartmentdefinition-patient.3.0.2.json` files come from. If they are easy to find on the FHIR website, maybe we don't need to commit them in git

Reply:
+1 to adding comments

The json definition file is easy to get for v4 on FHIR website, however, for v3, I had to manually clean up two resource type paths(these two gave dotted paths while other resource gave search parameters) before it can be feed into the script. I'll leave the two files in git and specify what's changed for v3 file.